src/db/car/forest.rs

```diff
@@ -149,16 +166,20 @@ impl<ReaderT: super::RandomAccessFileReader> ForestCar<ReaderT> {
         Ok((header, footer))
     }
 
-    pub fn roots(&self) -> &NonEmpty<Cid> {
-        &self.roots
+    pub fn head_tipset_key(&self) -> &NonEmpty<Cid> {
```

Comment:
Let's have some documentationor comments around it, pointing to the relevant FRC for more details.

Reply:
Fixed

Reply:
Fixed.